.github/workflows/test_flavor_qemu.yml

```diff
@@ -99,6 +106,13 @@ jobs:
             echo "no raw or pxe file, QEMU test failed"
             exit 1
           fi
+      - name: Save EDK2 cache
+        id: tests-ng-edk2-cache-save
+        uses: actions/cache/save@0057852bfaa89a56745cba8c7296529d2fc39830 # pin@v4.3.0
+        with:
+          path: 
+            .build/edk2-*
+          key: tests-ng-edk2-${{ hashFiles('tests-ng/util/build.makefile')}}
```

Comment:
I do not think we need this.

https://github.com/gardenlinux/gardenlinux/pull/3502/files#diff-522b408fac8deafb9e5429fc5a54cafb9697645fd31bfb18852f5cab16752cfcR39-R43 already puts the test distribution in the correct path `tests-ng/.build`.

Restoring this should actually already cache the `edk2-*` files.